### ISSUE TITLE
TypeError at /assets/service/add/

### ISSUE THREAD

**Issue by renki81** [allegro/ralph#2255]
Hi
I become this error when i will add a new Service:

```
TypeError at /assets/service/add/
'NoneType' object is not iterable
Request Method: GET
Request URL:    http://myip/assets/service/add/?_popup=1&_changelist_filters=_to_field%3Did%26_popup%3D1&_to_field=id
Django Version: 1.8.9
Exception Type: TypeError
Exception Value:    
'NoneType' object is not iterable
Exception Location: /opt/ralph/ralph-core/lib/python3.4/site-packages/ralph/admin/widgets.py in render, line 234
Python Executable:  /opt/ralph/ralph-core/bin/python
Python Version: 3.4.3
Python Path:    
['/opt/ralph/ralph-core/bin',
 '/opt/ralph/ralph-core/lib/python3.4',
 '/opt/ralph/ralph-core/lib/python3.4/plat-x86_64-linux-gnu',
 '/opt/ralph/ralph-core/lib/python3.4/lib-dynload',
 '/usr/lib/python3.4',
 '/usr/lib/python3.4/plat-x86_64-linux-gnu',
 '/opt/ralph/ralph-core/lib/python3.4/site-packages']
Server time:    Mon, 29 Feb 2016 18:25:33 +0100
Error during template rendering

In template /opt/ralph/ralph-core/lib/python3.4/site-packages/ralph/lib/foundation/templates/foundation_form/form_field.html, error at line 14


'NoneType' object is not iterable
4     <div class="row">
5       <div class="small-3 columns">
6         {% if show_label %}
7           {% label field %}
8         {% endif %}
9       </div>
10      <div class="small-9 end columns {% if field.errors %} error{% endif %}">
11        {% if field.is_readonly %}
12          <span class='read-only'>{{ field.contents|default:"-" }}</span>
13        {% else %}
14  

      {{ admin_field }}



15        {% endif %}
16        {% if admin_field.errors %}
17          {% for error in admin_field.errors %}
18            <small class="error">{{ error }}</small>
19          {% endfor %}
20        {% endif %}
21        {% if admin_field.help_text %}
22          <span class="help-text">
23            <i
24              class="fa fa-lg fa-info-circle has-tip tip-top"


Environment:


Request Method: GET
Request URL: http://myip/assets/service/add/?_popup=1&_changelist_filters=_to_field%3Did%26_popup%3D1&_to_field=id

Django Version: 1.8.9
Python Version: 3.4.3
Installed Applications:
('ralph.admin',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.humanize',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'import_export',
 'mptt',
 'reversion',
 'sitetree',
 'ralph.accounts',
 'ralph.assets',
 'ralph.attachments',
 'ralph.back_office',
 'ralph.data_center',
 'ralph.licences',
 'ralph.domains',
 'ralph.supports',
 'ralph.security',
 'ralph.lib.foundation',
 'ralph.lib.table',
 'ralph.data_importer',
 'ralph.dc_view',
 'ralph.reports',
 'ralph.virtual',
 'ralph.operations',
 'ralph.lib.transitions',
 'ralph.lib.permissions',
 'rest_framework',
 'rest_framework.authtoken',
 'taggit',
 'taggit_serializer')
Installed Middleware:
('django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware')


Template error:
In template /opt/ralph/ralph-core/lib/python3.4/site-packages/ralph/lib/foundation/templates/foundation_form/form_field.html, error at line 14
   'NoneType' object is not iterable

   4 :   <div class="row">



   5 :     <div class="small-3 columns">



   6 :       {% if show_label %}



   7 :         {% label field %}



   8 :       {% endif %}



   9 :     </div>



   10 :     <div class="small-9 end columns {% if field.errors %} error{% endif %}">



   11 :       {% if field.is_readonly %}



   12 :         <span class='read-only'>{{ field.contents|default:"-" }}</span>



   13 :       {% else %}



   14 :          {{ admin_field }} 



   15 :       {% endif %}



   16 :       {% if admin_field.errors %}



   17 :         {% for error in admin_field.errors %}



   18 :           <small class="error">{{ error }}</small>



   19 :         {% endfor %}



   20 :       {% endif %}



   21 :       {% if admin_field.help_text %}



   22 :         <span class="help-text">



   23 :           <i



   24 :             class="fa fa-lg fa-info-circle has-tip tip-top"


Traceback:
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/core/handlers/base.py" in get_response
  164.                 response = response.render()
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/response.py" in render
  158.             self.content = self.rendered_content
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/response.py" in rendered_content
  135.         content = template.render(context, self._request)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/backends/django.py" in render
  74.         return self.template.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  210.                     return self._render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/loader_tags.py" in render
  135.         return compiled_parent._render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/loader_tags.py" in render
  135.         return compiled_parent._render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/loader_tags.py" in render
  65.                 result = block.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/loader_tags.py" in render
  65.                 result = block.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/defaulttags.py" in render
  217.                             nodelist.append(node.render(context))
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/loader_tags.py" in render
  159.                 return template.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  212.                 return self._render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/defaulttags.py" in render
  217.                             nodelist.append(node.render(context))
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/defaulttags.py" in render
  217.                             nodelist.append(node.render(context))
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  1294.                     return t.render(new_context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  212.                 return self._render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/defaulttags.py" in render
  329.                 return nodelist.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/template/debug.py" in render
  92.             output = force_text(output)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/utils/encoding.py" in force_text
  90.                     s = six.text_type(s)
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/utils/html.py" in <lambda>
  399.         klass.__str__ = lambda self: mark_safe(klass_str(self))
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/forms/forms.py" in __str__
  537.         return self.as_widget()
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/django/forms/forms.py" in as_widget
  593.         return force_text(widget.render(name, self.value(), attrs=attrs))
File "/opt/ralph/ralph-core/lib/python3.4/site-packages/ralph/admin/widgets.py" in render
  234.                 force_text(v) for v in value

Exception Type: TypeError at /assets/service/add/
Exception Value: 'NoneType' object is not iterable
```

**Comment by xcezzz** [comment]
Just setup a development environment today to set this up, and receiving this too.

**Comment by ar4s** [comment]
Fixed in #2258
